### ISSUE TITLE
Logfiles swamped with error messages for HS100

### ISSUE THREAD

**Issue by LightIsLife** [computerlyrik/openhab2-addon-hs110#2]
Thanks again for making the .jar available.
I have several HS100, the binding picks them up, PaperUI lists them as unknown device:
`HS110 at 192.168.1.200
Unkown device
hs110:HS110:80067F723217A97B50E5202233DA6EC3178F65C0
`

However, after discovery of all 6 devices that we have, the logfile gets swamped with the error message below (several messages per second)
2017-05-06 17:40:22.042 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006AA465BF43CF0896F30871D6973A8178F837C' to inbox.
2017-05-06 17:40:22.051 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006E0FB724AC7D803F6B4BDFC427F51178EFA2C' to inbox.
2017-05-06 17:40:22.058 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006E80708F4068A8149ABA2F8BE4CA9178EA7D3' to inbox.
2017-05-06 17:40:22.066 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006DE9FFCC783765F707B7FCCA18FE4178FCB47' to inbox.
2017-05-06 17:40:22.074 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006E074D6A381F4F1060150A814DE57178EB5A8' to inbox.
2017-05-06 17:40:22.081 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:80067F723217A97B50E5202233DA6EC3178F65C0' to inbox.
2017-05-06 17:40:22.233 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-06 17:40:22.384 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-06 17:40:22.535 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-06 17:40:22.687 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-06 17:40:22.839 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-06 17:40:22.990 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time

**Comment by computerlyrik** [comment]
The discovery service polls for new UDP answers from existing Plugs. This will last few seconds only, correct?
I rewrote the message and placed it to DEBUG to prevent irritation. This is totally ok.

Please make sure you enable DEBUG for this plugin when posting LOGs. Thank you!

**Does anything work on your HS100?** I will have a look for the namings, to get the "Unknown Device" out of the PaperUI.

**Comment by computerlyrik** [comment]
Yes, for further investigation i need a DEBUG log. plase see http://docs.openhab.org/administration/logging.html
The package to enable debug for would be `org.openhab.binding.hs110`

**Comment by LightIsLife** [comment]
Thanks so much. Here the awesome news: the plugs react! Thank you.
Actually the "no response received" messages go on for minutes (more than 5 min), here some of the logs (debug mode), this is the output after discovering new things specific to the HS110 binding through PaperUI:
```
2017-05-26 19:49:57.300 [INFO ] [.astro.internal.job.AbstractDailyJob] - Scheduled astro event-jobs for thing astro:sun:home
2017-05-26 19:51:50.853 [DEBUG] [org.openhab.binding.hs110           ] - BundleEvent INSTALLED - org.openhab.binding.hs110
2017-05-26 19:51:51.011 [DEBUG] [org.openhab.binding.hs110           ] - BundleEvent RESOLVED - org.openhab.binding.hs110
2017-05-26 19:51:51.041 [DEBUG] [org.openhab.binding.hs110           ] - BundleEvent STARTING - org.openhab.binding.hs110
2017-05-26 19:51:51.046 [DEBUG] [org.openhab.binding.hs110           ] - BundleEvent STARTED - org.openhab.binding.hs110
2017-05-26 19:51:51.090 [DEBUG] [org.openhab.binding.hs110           ] - ServiceEvent REGISTERED - {org.eclipse.smarthome.config.discovery.DiscoveryService}={component.name=org.openhab.binding.hs110.discovery.HS110DiscoveryService, component.id=215, service.id=363, service.bundleid=211, service.scope=bundle} - org.openhab.binding.hs110
2017-05-26 19:51:51.099 [DEBUG] [org.openhab.binding.hs110           ] - ServiceEvent REGISTERED - {org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory}={component.name=binding.hs110, component.id=216, service.id=364, service.bundleid=211, service.scope=bundle} - org.openhab.binding.hs110
2017-05-26 19:52:30.669 [DEBUG] [s110.discovery.HS110DiscoveryService] - Stop Scan for hs110:HS110 hs110:HS100
2017-05-26 19:52:30.678 [DEBUG] [s110.discovery.HS110DiscoveryService] - Start Scan for hs110:HS110 hs110:HS100
2017-05-26 19:52:30.843 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-26 19:52:30.915 [DEBUG] [s110.discovery.HS110DiscoveryService] - Discovery returned package with length 569
2017-05-26 19:52:30.918 [DEBUG] [s110.discovery.HS110DiscoveryService] - Detecting HS110 by data: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:91:4D","deviceId":"8006AA465BF43CF0896F30871D6973A8178F837C","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Closet Light","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-47,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:52:30.924 [DEBUG] [s110.discovery.HS110DiscoveryService] - HS110 with id 8006AA465BF43CF0896F30871D6973A8178F837C found on 192.168.1.205 
2017-05-26 19:52:30.930 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006AA465BF43CF0896F30871D6973A8178F837C' to inbox.
2017-05-26 19:52:30.943 [DEBUG] [s110.discovery.HS110DiscoveryService] - Discovery returned package with length 569
2017-05-26 19:52:30.946 [DEBUG] [s110.discovery.HS110DiscoveryService] - Detecting HS110 by data: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:89:37","deviceId":"8006E0FB724AC7D803F6B4BDFC427F51178EFA2C","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Router Light","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-22,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:52:30.948 [DEBUG] [s110.discovery.HS110DiscoveryService] - HS110 with id 8006E0FB724AC7D803F6B4BDFC427F51178EFA2C found on 192.168.1.202 
2017-05-26 19:52:31.015 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006E0FB724AC7D803F6B4BDFC427F51178EFA2C' to inbox.
2017-05-26 19:52:31.026 [DEBUG] [s110.discovery.HS110DiscoveryService] - Discovery returned package with length 570
2017-05-26 19:52:31.028 [DEBUG] [s110.discovery.HS110DiscoveryService] - Detecting HS110 by data: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:92:6F","deviceId":"8006B53B2B534C730B173999EB58B38D178FE547","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Markus' Light","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-59,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:52:31.029 [DEBUG] [s110.discovery.HS110DiscoveryService] - HS110 with id 8006B53B2B534C730B173999EB58B38D178FE547 found on 192.168.1.207 
2017-05-26 19:52:31.038 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006B53B2B534C730B173999EB58B38D178FE547' to inbox.
2017-05-26 19:52:31.039 [DEBUG] [s110.discovery.HS110DiscoveryService] - Discovery returned package with length 570
2017-05-26 19:52:31.041 [DEBUG] [s110.discovery.HS110DiscoveryService] - Detecting HS110 by data: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:89:33","deviceId":"8006E074D6A381F4F1060150A814DE57178EB5A8","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Drawers Light","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-56,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:52:31.042 [DEBUG] [s110.discovery.HS110DiscoveryService] - HS110 with id 8006E074D6A381F4F1060150A814DE57178EB5A8 found on 192.168.1.204 
2017-05-26 19:52:31.049 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006E074D6A381F4F1060150A814DE57178EB5A8' to inbox.
2017-05-26 19:52:31.050 [DEBUG] [s110.discovery.HS110DiscoveryService] - Discovery returned package with length 570
2017-05-26 19:52:31.051 [DEBUG] [s110.discovery.HS110DiscoveryService] - Detecting HS110 by data: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:91:8E","deviceId":"80060C32F1FE38116D16A1AE3AE4638C178F9D14","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Terri's Light","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-55,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:52:31.053 [DEBUG] [s110.discovery.HS110DiscoveryService] - HS110 with id 80060C32F1FE38116D16A1AE3AE4638C178F9D14 found on 192.168.1.206 
2017-05-26 19:52:31.059 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:80060C32F1FE38116D16A1AE3AE4638C178F9D14' to inbox.
2017-05-26 19:52:31.060 [DEBUG] [s110.discovery.HS110DiscoveryService] - Discovery returned package with length 568
2017-05-26 19:52:31.061 [DEBUG] [s110.discovery.HS110DiscoveryService] - Detecting HS110 by data: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:90:78","deviceId":"80067F723217A97B50E5202233DA6EC3178F65C0","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Office Lamp","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-78,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:52:31.062 [DEBUG] [s110.discovery.HS110DiscoveryService] - HS110 with id 80067F723217A97B50E5202233DA6EC3178F65C0 found on 192.168.1.200 
2017-05-26 19:52:31.070 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:80067F723217A97B50E5202233DA6EC3178F65C0' to inbox.
2017-05-26 19:52:31.071 [DEBUG] [s110.discovery.HS110DiscoveryService] - Discovery returned package with length 568
2017-05-26 19:52:31.072 [DEBUG] [s110.discovery.HS110DiscoveryService] - Detecting HS110 by data: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:89:B5","deviceId":"8006E80708F4068A8149ABA2F8BE4CA9178EA7D3","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Piano Light","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-48,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:52:31.073 [DEBUG] [s110.discovery.HS110DiscoveryService] - HS110 with id 8006E80708F4068A8149ABA2F8BE4CA9178EA7D3 found on 192.168.1.203 
2017-05-26 19:52:31.080 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006E80708F4068A8149ABA2F8BE4CA9178EA7D3' to inbox.
2017-05-26 19:52:31.081 [DEBUG] [s110.discovery.HS110DiscoveryService] - Discovery returned package with length 567
2017-05-26 19:52:31.081 [DEBUG] [s110.discovery.HS110DiscoveryService] - Detecting HS110 by data: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:90:3C","deviceId":"8006DE9FFCC783765F707B7FCCA18FE4178FCB47","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Couch Lamp","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-64,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:52:31.083 [DEBUG] [s110.discovery.HS110DiscoveryService] - HS110 with id 8006DE9FFCC783765F707B7FCCA18FE4178FCB47 found on 192.168.1.201 
2017-05-26 19:52:31.090 [INFO ] [g.discovery.internal.PersistentInbox] - Added new thing 'hs110:HS110:8006DE9FFCC783765F707B7FCCA18FE4178FCB47' to inbox.
2017-05-26 19:52:31.241 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-26 19:52:31.393 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
[....and many more of those...]
```
a little while later (~ 1min) I had one of those:
```
2017-05-26 19:52:35.675 [DEBUG] [s110.discovery.HS110DiscoveryService] - Stop Scan for hs110:HS110 hs110:HS100
```
and many more no responses
I than tried to add a thing through the PaperUI and the following error repeats itself for more than 5 mins:
```
2017-05-26 19:55:01.484 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-26 19:55:01.502 [DEBUG] [org.openhab.binding.hs110           ] - ServiceEvent REGISTERED - {org.eclipse.smarthome.config.core.status.ConfigStatusProvider}={service.id=365, service.bundleid=211, service.scope=singleton} - org.openhab.binding.hs110
2017-05-26 19:55:01.517 [INFO ] [b.binding.hs110.handler.HS110Handler] - Initializing plug on ip 192.168.1.200
2017-05-26 19:55:01.523 [DEBUG] [b.binding.hs110.handler.HS110Handler] - Refreshing hs110:HS110:80067F723217A97B50E5202233DA6EC3178F65C0
2017-05-26 19:55:01.525 [DEBUG] [openhab.binding.hs110.internal.HS110] - Executing command {"system":{"get_sysinfo":null}}
2017-05-26 19:55:01.550 [DEBUG] [openhab.binding.hs110.internal.HS110] - Executing command {"emeter":{"get_realtime":null}}
2017-05-26 19:55:01.577 [DEBUG] [b.binding.hs110.handler.HS110Handler] - Updated energy Data for hs110:HS110:80067F723217A97B50E5202233DA6EC3178F65C0: {"emeter":{"err_code":-1,"err_msg":"module not support"}}
2017-05-26 19:55:01.578 [DEBUG] [b.binding.hs110.handler.HS110Handler] - Updated sysinfo Data for hs110:HS110:80067F723217A97B50E5202233DA6EC3178F65C0: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:90:78","deviceId":"80067F723217A97B50E5202233DA6EC3178F65C0","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Office Lamp","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-79,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:55:01.578 [DEBUG] [openhab.binding.hs110.internal.HS110] - Executing command {"system":{"get_sysinfo":null}}
2017-05-26 19:55:01.602 [DEBUG] [openhab.binding.hs110.internal.HS110] - Executing command {"emeter":{"get_realtime":null}}
2017-05-26 19:55:01.617 [DEBUG] [b.binding.hs110.handler.HS110Handler] - Updated energy Data for hs110:HS110:80067F723217A97B50E5202233DA6EC3178F65C0: {"emeter":{"err_code":-1,"err_msg":"module not support"}}
2017-05-26 19:55:01.618 [DEBUG] [b.binding.hs110.handler.HS110Handler] - Updated sysinfo Data for hs110:HS110:80067F723217A97B50E5202233DA6EC3178F65C0: {"system":{"get_sysinfo":{"err_code":0,"sw_ver":"1.1.0 Build 160503 Rel.145047","hw_ver":"1.0","type":"IOT.SMARTPLUGSWITCH","model":"HS100(EU)","mac":"50:C7:BF:06:90:78","deviceId":"80067F723217A97B50E5202233DA6EC3178F65C0","hwId":"22603EA5E716DEAEA6642A30BE87AFCA","fwId":"BEF3C9ECA17BF143A7D627AB2D1382C4","oemId":"812A90EB2FCF306A993FAD8748024B07","alias":"Office Lamp","dev_name":"Wi-Fi Smart Plug","icon_hash":"","relay_state":0,"on_time":0,"active_mode":"schedule","feature":"TIM","updating":0,"rssi":-81,"led_off":0,"latitude":41.825733,"longitude":12.488388}}}
2017-05-26 19:55:01.622 [DEBUG] [b.binding.hs110.handler.HS110Handler] - Exception occurred during execution
java.lang.NullPointerException
	at org.openhab.binding.hs110.internal.HS110.parseWattage(HS110.java:120)[211:org.openhab.binding.hs110:2.1.0.201705041100]
	at org.openhab.binding.hs110.handler.HS110Handler.getWattage(HS110Handler.java:188)[211:org.openhab.binding.hs110:2.1.0.201705041100]
	at org.openhab.binding.hs110.handler.HS110Handler.access$5(HS110Handler.java:186)[211:org.openhab.binding.hs110:2.1.0.201705041100]
	at org.openhab.binding.hs110.handler.HS110Handler$1.run(HS110Handler.java:135)[211:org.openhab.binding.hs110:2.1.0.201705041100]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)[:1.8.0_131]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)[:1.8.0_131]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)[:1.8.0_131]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)[:1.8.0_131]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)[:1.8.0_131]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)[:1.8.0_131]
	at java.lang.Thread.run(Thread.java:748)[:1.8.0_131]
2017-05-26 19:55:01.638 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-26 19:55:01.790 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
2017-05-26 19:55:01.942 [INFO ] [s110.discovery.HS110DiscoveryService] - No response received in time
```

Hope that helps!

**Comment by LightIsLife** [comment]
apologies, closed by accident

**Comment by computerlyrik** [comment]
Closed in favor of openhab/openhab2-addons#2348 - new rewritten Plugin, new Author
